trunk/source/CACSL2BoogieTranslator/src/de/uni_freiburg/informatik/ultimate/cdt/translation/implementation/base/chandler/StaticObjectsHandler.java

```diff
@@ -85,14 +85,11 @@ public class StaticObjectsHandler {
 	private final Map<TypeDeclaration, CDeclaration> mTypeDeclarationToCDeclaration;
 	private final Map<String, TypeDeclaration> mIncompleteType2TypeDecl;
 
-	private final ILogger mLogger;
-
 	public StaticObjectsHandler(final ILogger logger) {

```

Comment:
Since the `ILogger` member is removed from the class, the unused parameter in the constructor can also be removed.

```suggestion
	public StaticObjectsHandler() {
```

Reply:
See [17dd19a](https://github.com/ultimate-pa/ultimate/pull/756/commits/17dd19afa57a31fa314c65c59b02ab8c65618db3)